.github/buildomat/build-and-test.sh

```diff
@@ -53,7 +53,8 @@ banner ls-apis
     source ./tools/include/force-git-over-https.sh;
     ptime -m cargo xtask ls-apis apis &&
         ptime -m cargo xtask ls-apis deployment-units &&
-        ptime -m cargo xtask ls-apis servers
+        ptime -m cargo xtask ls-apis servers &&
+        ptime -m cargo xtask ls-apis dag-check
```

Comment:
Super nit:

> cargo xtask ls-apis dag-check

Could this just be `check`, or `check-dag`? I personally like anything that does a check have its name start with `check`.

Reply:
Okay, I clearly overthought this.  I'm trying to resolve this (useful!) guideline with a couple of other rules/guidelines I use:

- I usually use the `subject-verb[-object]` naming convention (hence "dag-check"), both in code and shell command names.  This is pretty arbitrary but I do find it results in more discoverable code/commands because related things are grouped together.  This would be a bigger deal if we ever add other `dag` subcommands (i.e., I think `dag-check` and `dag-display` is much better than `check-dag` and `display-dag`, and also lends itself to `dag check` / `dag display`).
- I follow clap's default of having the Rust struct/function names match the command names so it's easy to find the source for a command.

So if we change this to `check-dag`: (1) this gets awkward if we add other dag commands later, and (2) either we change all the Rust names also or we don't, but either way, they diverge from some of these conventions.

Like I said, I'm overthinking this.

But what if we just call this `cargo xtask ls-apis check` on the outside?  I'd have the command (the names in src/bin/ls-apis.rs) call it "check" but the library interface would still call it `dag_check()`/`DagCheck`.  If we add other checks in the future, we'd probably want them done by `ls-apis check` anyway.

Reply:
I went ahead and did this in 26bbf1c.

Reply:
sounds good!